jekyll/assets/css/main.scss

```diff
@@ -390,3 +390,10 @@ pre {
 .serveronly{
 	color: #4e9eb3; 
   }
+
+.table-migrate td, .table-migrate th {
+  width: 50%;
+  display: inline-block;
+  height: 100%;
+}
+
```

Comment:
Nice work on this. Looks great on desktop. How do you feel about dropping the two into a column view on mobile?

![image](https://user-images.githubusercontent.com/12987958/74459151-d1abb080-4e58-11ea-98ad-541b4199a06a.png)